data/map.txt

```diff
@@ -27873,6 +31830,13 @@ planet "Ki Patek Ka"
 	shipyard Kimek
 	outfitter "Coalition Advanced"
 
+planet Kiteg-hamlior-tancla
+	attributes tourism urban xapleaux
+	landscape land/dmottl2
+	description "This quite dry and dusty world is dominated by chaparral ecosystems, and often as not rain is the result of smoke in the atmosphere instead of merely the accumulation of water vapor. The ground tremors frequently on account of its very large stellar neighbor: a moon nearly half its own mass. As a consequence Xapleaux structures are built well into the ground to rest upon bedrock instead of shifting soils, with an even greater preponderance of crystal skylights than normal to transmit the light of The Brilliance down into lower levels."
+	spaceport `Plastics and medicines, medicinal plastics, plastics in medicines, and medicines in plastic. These are the exports of this world which has buried more vegetation than most worlds have ever grown. The Xapleaux here run the gamut of the more extremely bizarre looking variants, which dominate the population; so much so that the spaceport has multiple "burning" stations to accommodate the rarest variety of Xapleaux that can frequently be seen desperately waddling into the mirrored glass chamber and promptly catching fire. After a few moments of billowing flame they emerge, shaking off ash and looking much less burdened with the weight of their overgrown fronds burned away.`
```

Comment:
No, the point is that this is home to almost exclusively, just the most bizarre varieties.
```suggestion
	spaceport `Plastics and medicines, medicinal plastics, plastics in medicines, and medicines in plastic. These are the exports of this world which has buried more vegetation than most worlds have ever grown. The the full gamut of the uncommon and extremely bizarre looking variants of Xapleaux dominate the local population here; so much so that the spaceport has multiple "burning" stations to accommodate the rarest variety of Xapleaux that can frequently be seen desperately waddling into the mirrored glass chamber and promptly catching fire. After a few moments of billowing flame they emerge, shaking off ash and looking much less burdened with the weight of their overgrown fronds burned away.`
```